cmd/tailscale/cli/utils.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
```

Comment:
Blocking: We don’t normally introduce utils.go files. This single utility can go in cmd/tailscale/cli/cli.go, alongside the other utilities like `outf` and `fatalf`.